settings.gradle

```diff
@@ -46,6 +46,7 @@ include 'display-information:add-graphics-with-renderer',
         'search:find-place',
         'search:geocode-online',
         'search:reverse-geocode-online',
+        'symbology:symbol-dictionary-sample',
```

Comment:
Try to keep these in alphabetical order. The IDEs show it that way. Makes it easier to find.